### PR TITLE
Add CVE-2021-4175 for 8a7d16e0-9a46-4710-a029-c89c33c01528

### PR DESCRIPTION
**Pull request by JamieSlome**
Add CVE-2021-4175 for [8a7d16e0-9a46-4710-a029-c89c33c01528](https://huntr.dev/bounties/8a7d16e0-9a46-4710-a029-c89c33c01528) @huntr-helper

### CODE REVIEW

File: 2021/4xxx/CVE-2021-4175.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
+{
```
Comment (cve-team) [inline 2021/4xxx/CVE-2021-4175.json:1]: None is not of type 'string'

**Review by cve-team — CHANGES_REQUESTED** [review]
Auto validation errors:

**Comment by anthnysingleton** [comment]
@JamieSlome Please resolution the validation issues and the merge conflict. Once resolved we can process and merge request.

**Comment by JamieSlome** [comment]
@anthnysingleton - sorted! ❤️

**Pull request merged**